### PR TITLE
Adds ERP Squads

### PR DESCRIPTION
**Pull request by Incoming5643**
Spawned in their underpants with a dumb name, requisite floorlength braids, and pretty much nothing else. They're here to party and/or manage to kill everyone somehow anyways.

Their objectives are to be sexy but not get banned for being TOO sexy

### CODE REVIEW

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -71,6 +71,11 @@
 				log_admin("[key_name(usr)] created a emergency response team.")
 				if(!src.makeEmergencyresponseteam())
 					usr << "<span class='danger'>Unfortunatly there were not enough candidates available.</span>"
+			if("14")
+				message_admins("[key_name(usr)] created an ERP squad.")
+				log_admin("[key_name(usr)] created an ERP squad.")
+				if(!src.makeERPsquad())
+					usr << "<span class='danger'>Fortunatly there were not enough candidates available.</span>"
```
Comment (optimumtact) [inline code/modules/admin/topic.dm]: fortunately

File: code/modules/admin/verbs/one_click_antag.dm [hunk 2]
```diff
@@ -522,6 +523,91 @@ client/proc/one_click_antag()
...
+			switch(alert(G,"Do you wish to be considered for an elite Nanotrasen ERP Squad being sent in?","Please answer in 30 seconds!","Yes","No"))
+				if("Yes")
+					if((world.time-time_passed)>300)//If more than 30 game seconds passed.
+						return
+					candidates += G
+				if("No")
+					return
+				else
+					return
+	sleep(300)
```
Comment (optimumtact) [inline code/modules/admin/verbs/one_click_antag.dm]: Jesus christ we need a better way to poll people about things than this sleep shit

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (Incoming5643) [inline code/modules/admin/topic.dm]: Hey don't blame me for this one, the guy above me I copied it from fucked it up too

File: code/modules/admin/verbs/one_click_antag.dm (hunk 2 above)
Comment (Incoming5643) [inline code/modules/admin/verbs/one_click_antag.dm]: BE_ERPER flag

File: code/modules/admin/verbs/one_click_antag.dm [hunk 3]
```diff
@@ -522,6 +523,91 @@ client/proc/one_click_antag()
...
...,"Skye","Sloane","Smoky","Soleil","Sonia","Song","Sorority","Spice","Spider","Spring","Staci","Stacia","Starr","Stevie","Storm","Stormy","Strawberry","Suavecita","Sublime","Sugar","Summer","Sundae","Sundance","Sunday","Sunni","Sunny","Sunshine","Sunset","Susanna","Suzanne","Swallow","Sweet","Sweetness","Tabitha","Taffy","Talia","Tallulah","Tamar","Tamara","Tammi","Tane","Tanya","Tara","Tasha","Tasty","Tatiana","Tawni","Taylor","Temper","Tempest","Temptation","Terra","Terror","Tess","Tessa","Texxxas","Thai","Thia","Thumper","Thyme","Tia","Tickle","Tiffany","Tiger","Tigra","Tika","Time","Tina","Tinker","Tisha","Toffee","Tommy","Toni","Topaz","Tori","Traci","Tracy","Tricia","Trinity","Trip","Tristana","Trixie","Trouble","Tuesday","Tyffany","Tyler","Tyra","Unique","Utopia","Valentina","Valentine","Vandal","Vanessa","Vanity","Velvet","Venus","Veronica","Viagra","Victoria","Victory","Viola","Violet","Viper","Virginia","Virgo","Vision","Vivienne","Vixen","Vonda","Wanda","Wednesday","Wendy","Whimsy","Whisky","Whisper","Willow","Windy","Winter","Wish","Wyndi","Xanadu","Xanthe","Xaviera","Xena","Xiola","Xtase","Yasmin","Yolanda","Yvette","Yvonne","Zena","Zenith","Zinnia","Zoe","Zoey","Zora")
```
Comment (optimumtact) [inline code/modules/admin/verbs/one_click_antag.dm]: 10/10 but move this up to global so we can reuse it

File: code/game/gamemodes/objective.dm [hunk 4]
```diff
@@ -575,3 +575,12 @@ datum/objective/destroy/update_explanation_text()
 		explanation_text = "Destroy [target.name], the experimental AI."
 	else
 		explanation_text = "Free Objective"
+
+datum/objective/dontgetbanned
+	dangerrating = 50000
```
Comment (Iamgoofball) [inline code/game/gamemodes/objective.dm]: >danger rating isnt 69

**Comment by Dorsisdwarf** [comment]
Spawn them with a bottle of rum.

**Comment by tedward1337** [comment]
Bring this back 2016

**Comment by Ricotez** [comment]
how many merge conflicts will it have after a year

**Comment by Incoming5643** [comment]
Considering it's all new lines, I'm thinking zero or maybe one or two if someone's renamed a file

**Comment by Dorsisdwarf** [comment]
all

**Pull request closed without merging**